### ISSUE TITLE
PFHC02 Issues

### ISSUE THREAD

**Issue by StellarExile** [KOTORCommunityPatches/TSL_Community_Patch#51]
Something I've noticed about the head PFHC02 is that a pretty noticeable gap in its neck can sometimes be seen when the camera is facing the back of the head. I noticed it especially when I did a LS Fem playthrough of TSL. I'm suspecting that it might be an issue with the neck vertex alignment if my language is correct.

**Comment by DarthParametric** [comment]
Looks like the rear-most vert is weighted incorrectly. It has about 50% weighting to `neck_g`, rather than 100% to `necklwr_g` (effectively the torso). This will cause it to lift away from the body with any forward inclination of the head. It's an easy fix though.

**Comment by StellarExile** [comment]
That's a relief. Is there anything that can be done?

**Comment by DarthParametric** [comment]
We'll address it in the next release.

**Comment by StellarExile** [comment]
Thank you! Is it okay if I post some more issues that I have found?

**Comment by DarthParametric** [comment]
Of course, that's what Issues are for. Although we'll only address those we deem are within the Community Patch's remit.
